deepmd/pt/modifier/dipole_charge.py

```
@@ -0,0 +1,351 @@
+    model_charge_map
+            Gives the amount of charge for the wfcc
+    sys_charge_map
+            Gives the amount of charge for the real atoms
+    ewald_h
+            Grid spacing of the reciprocal part of Ewald sum. Unit: A
+    ewald_beta
+            Splitting parameter of the Ewald sum. Unit: A^{-1}
+    """
+
+    def __new__(
+        cls, *args: tuple, model_name: str | None = None, **kwargs: dict
```

Ruff (0.14.10)

Unused static method argument: `args`

(ARG004)